Jellyfin.Server/Properties/launchSettings.json

```diff
@@ -2,12 +2,18 @@
   "profiles": {
     "Jellyfin.Server": {
       "commandName": "Project",
+      "launchBrowser": true,
+      "launchUrl": "web",
+      "applicationUrl": "http://localhost:8096",
       "environmentVariables": {
         "ASPNETCORE_ENVIRONMENT": "Development"
       }
     },
     "Jellyfin.Server (nowebclient)": {
       "commandName": "Project",
+      "launchBrowser": true,
+      "launchUrl": "api-docs/swagger",
```

Comment:
This profile used to not launch a web browser but now it launches the API docs. Maybe you could edit this profile to not launch the browser and add a seperate profile to launch the docs. But this may just be my personal preference.

Reply:
My mistake - there was code in startup wizard to load the API docs if the web client wasn't being hosted. I assumed that should be the default behavior. I'll add a separate profile for the API docs and modify this profile to not launch the browser at all.